includes/class-algolia-cli.php

```diff
@@ -68,9 +71,10 @@ public function reindex( $args, $assoc_args ) {
 			WP_CLI::error( 'The configuration for this website does not allow to contact the Algolia API.' );
 		}
 
-		$index_id = isset( $args[0] ) ? $args[0] : null;
-		$clear    = WP_CLI\Utils\get_flag_value( $assoc_args, 'clear' );
-		$all      = WP_CLI\Utils\get_flag_value( $assoc_args, 'all' );
+		$index_id  = isset( $args[0] ) ? $args[0] : null;
+		$clear     = WP_CLI\Utils\get_flag_value( $assoc_args, 'clear' );
+		$all       = WP_CLI\Utils\get_flag_value( $assoc_args, 'all' );
+		$from_page = isset( $assoc_args['from_page'] ) ? intval( $assoc_args['from_page'] ) ? 1;
```

Comment:
We should change this line to use `WP_CLI\Utils\get_flag_value` function like above. 

Here is the documentation for the function: https://make.wordpress.org/cli/handbook/references/internal-api/wp-cli-utils-get-flag-value/

Reply:
Implemented now. Don't know why I did not use in the first place. I wrapped it in a `intval()`. Don't know if you'd prefer to do that later in the code - if so, let me know :)